packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -482,7 +490,7 @@ export class TokenRatesController extends PollingControllerV1<
       (obj, [tokenContractAddress, tokenPrice]) => {
         return {
           ...obj,
-          [toChecksumHexAddress(tokenContractAddress)]: tokenPrice.value,
+          [tokenContractAddress]: tokenPrice.value,
```

Comment:
By the time we get here, `tokenContractAddress` is already normalized (thanks to `#getTokenAddresses`).